Code/Engine/Foundation/Types/ArrayPtr.h

```diff
@@ -202,6 +202,7 @@ class ezArrayPtr
   }
 
   /// \brief Compares the two arrays for equality.
+  template <typename = typename std::enable_if<std::is_const<T>::value == false>>
```

Comment:
Needed to make this one mutually exclusive with the other function that also uses enable_if.